composition_libs/libraries/component/source.rb

```diff
@@ -73,15 +85,14 @@ def qty_display(round = 1, adj_quantities: false)
   # @param checkable [Boolean] whether to make the result checkable
   #   in a table
   # @return [Numeric, Hash]
-  def adjusted_qty(mult = 1.0, round = 1, checkable = true)
+  def adjusted_qty(mult = 1.0, round = 1, checkable = false)
     return if qty.nil?
     
-    adj_qty = (qty * mult).round(round)
+    @adj_qty = (qty * mult).round(round)
```

Comment:
This is undesirable for several reasons: First, there isn't a lot of value in setting an instance variable that copies the return of a method unless the computation that the method performs is expensive and you want to cache the result. Second, in having the method and variable that can return different results, you no longer have a single source of truth. Third, this looks like a pure function from the outside, but has a side effect of setting the instance variable. It's better to either set the variable or return the result of the computation, but not both.